app/src/main/res/raw/clash.yaml

```diff
@@ -0,0 +1,510 @@
+- DOMAIN-SUFFIX,qihucdn.com,DIRECT
+- DOMAIN-SUFFIX,qiniu.com,DIRECT
+- DOMAIN-SUFFIX,qiniucdn.com,DIRECT
+- DOMAIN-SUFFIX,qiyipic.com,DIRECT
+- DOMAIN-SUFFIX,qq.com,DIRECT
+- DOMAIN-SUFFIX,qqurl.com,DIRECT
+- DOMAIN-SUFFIX,rarbg.to,DIRECT
+- DOMAIN-SUFFIX,ruguoapp.com,DIRECT
+- DOMAIN-SUFFIX,segmentfault.com,DIRECT
+- DOMAIN-SUFFIX,sinaapp.com,DIRECT
+- DOMAIN-SUFFIX,smzdm.com,DIRECT
+- DOMAIN-SUFFIX,snapdrop.net,DIRECT
+- DOMAIN-SUFFIX,sogou.com,DIRECT
+- DOMAIN-SUFFIX,sogoucdn.com,DIRECT
+- DOMAIN-SUFFIX,sohu.com,DIRECT
+- DOMAIN-SUFFIX,soku.com,DIRECT
+- DOMAIN-SUFFIX,speedtest.net,DIRECT
+- DOMAIN-SUFFIX,sspai.com,DIRECT
+- DOMAIN-SUFFIX,suning.com,DIRECT
+- DOMAIN-SUFFIX,taobao.com,DIRECT
+- DOMAIN-SUFFIX,tencent.com,DIRECT
+- DOMAIN-SUFFIX,tenpay.com,DIRECT
+- DOMAIN-SUFFIX,tianyancha.com,DIRECT
+- DOMAIN-SUFFIX,tmall.com,DIRECT
+- DOMAIN-SUFFIX,tudou.com,DIRECT
+- DOMAIN-SUFFIX,umetrip.com,DIRECT
+- DOMAIN-SUFFIX,upaiyun.com,DIRECT
+- DOMAIN-SUFFIX,upyun.com,DIRECT
+- DOMAIN-SUFFIX,veryzhun.com,DIRECT
+- DOMAIN-SUFFIX,weather.com,DIRECT
+- DOMAIN-SUFFIX,weibo.com,DIRECT
+- DOMAIN-SUFFIX,xiami.com,DIRECT
+- DOMAIN-SUFFIX,xiami.net,DIRECT
+- DOMAIN-SUFFIX,xiaomicp.com,DIRECT
+- DOMAIN-SUFFIX,ximalaya.com,DIRECT
+- DOMAIN-SUFFIX,xmcdn.com,DIRECT
+- DOMAIN-SUFFIX,xunlei.com,DIRECT
+- DOMAIN-SUFFIX,yhd.com,DIRECT
+- DOMAIN-SUFFIX,yihaodianimg.com,DIRECT
+- DOMAIN-SUFFIX,yinxiang.com,DIRECT
+- DOMAIN-SUFFIX,ykimg.com,DIRECT
+- DOMAIN-SUFFIX,youdao.com,DIRECT
+- DOMAIN-SUFFIX,youku.com,DIRECT
+- DOMAIN-SUFFIX,zealer.com,DIRECT
+- DOMAIN-SUFFIX,zhihu.com,DIRECT
+- DOMAIN-SUFFIX,zhimg.com,DIRECT
+- DOMAIN-SUFFIX,zimuzu.tv,DIRECT
+
+# 抗 DNS 污染
+- DOMAIN-KEYWORD,amazon,Proxy
+- DOMAIN-KEYWORD,google,Proxy
+- DOMAIN-KEYWORD,gmail,Proxy
+- DOMAIN-KEYWORD,youtube,Proxy
+- DOMAIN-KEYWORD,facebook,Proxy
+- DOMAIN-SUFFIX,fb.me,Proxy
+- DOMAIN-SUFFIX,fbcdn.net,Proxy
+- DOMAIN-KEYWORD,twitter,Proxy
+- DOMAIN-KEYWORD,instagram,Proxy
+- DOMAIN-KEYWORD,dropbox,Proxy
+- DOMAIN-SUFFIX,twimg.com,Proxy
+- DOMAIN-KEYWORD,blogspot,Proxy
+- DOMAIN-SUFFIX,youtu.be,Proxy
+- DOMAIN-KEYWORD,whatsapp,Proxy
+
+# 常见广告域名屏蔽
+#- DOMAIN-KEYWORD,admarvel,REJECT
+#- DOMAIN-KEYWORD,admaster,REJECT
+#- DOMAIN-KEYWORD,adsage,REJECT
+#- DOMAIN-KEYWORD,adsmogo,REJECT
+#- DOMAIN-KEYWORD,adsrvmedia,REJECT
+#- DOMAIN-KEYWORD,adwords,REJECT
+#- DOMAIN-KEYWORD,adservice,REJECT
+#- DOMAIN-KEYWORD,domob,REJECT
+#- DOMAIN-KEYWORD,duomeng,REJECT
+#- DOMAIN-KEYWORD,dwtrack,REJECT
+#- DOMAIN-KEYWORD,guanggao,REJECT
+#- DOMAIN-KEYWORD,lianmeng,REJECT
+#- DOMAIN-SUFFIX,mmstat.com,REJECT
+#- DOMAIN-KEYWORD,omgmta,REJECT
+#- DOMAIN-KEYWORD,openx,REJECT
+#- DOMAIN-KEYWORD,partnerad,REJECT
+#- DOMAIN-KEYWORD,pingfore,REJECT
+#- DOMAIN-KEYWORD,supersonicads,REJECT
+#- DOMAIN-KEYWORD,tracking,REJECT
+#- DOMAIN-KEYWORD,uedas,REJECT
+#- DOMAIN-KEYWORD,umeng,REJECT
+#- DOMAIN-KEYWORD,usage,REJECT
+#- DOMAIN-KEYWORD,wlmonitor,REJECT
+#- DOMAIN-KEYWORD,zjtoolbar,REJECT
+
+# 国外网站
```

Comment:
I guess we don't have to maintain this foreign website domain list, do we?

Reply:
It would help when the IP could not be found in GeoIP database.

Reply:
OK.